app/src/main/java/com/asfoundation/wallet/ui/iab/LocalPaymentInteractor.kt

```diff
@@ -44,30 +44,29 @@ class LocalPaymentInteractor(private val deepLinkRepository: InAppDeepLinkReposi
         }
   }
 
-  fun getTransaction(uri: Uri): Observable<Transaction> {
-    return inAppPurchaseInteractor.getTransaction(uri.lastPathSegment)
-        .filter { isEndingState(it.status, it.type) }
-        .distinctUntilChanged { transaction -> transaction.status }
-  }
+  fun getTransaction(uri: Uri): Observable<Transaction> =
+      inAppPurchaseInteractor.getTransaction(uri.lastPathSegment)
+          .filter { isEndingState(it.status, it.type) }
+          .distinctUntilChanged { transaction -> transaction.status }
 
-  private fun isEndingState(status: Transaction.Status, type: String): Boolean {
-    return (status == PENDING_USER_PAYMENT && type == "TOPUP") || (status == COMPLETED && (type == "INAPP" || type == "INAPP_UNMANAGED")) || status == FAILED || status == CANCELED || status == INVALID_TRANSACTION
-  }
+  private fun isEndingState(status: Transaction.Status, type: String) =
+      (status == PENDING_USER_PAYMENT && type == "TOPUP") ||
+          (status == COMPLETED && (type == "INAPP" || type == "INAPP_UNMANAGED")) ||
+          status == FAILED ||
+          status == CANCELED ||
+          status == INVALID_TRANSACTION
```

Comment:
i don't think we gain anything from using the "=" here. I think we should only use these on simplier functions. the brackets helps to have a better visualization on where the method starts and ends. We should discuss this as team maybe